2018-10-17 16:49:26	Client initialized
2018-10-17 16:49:29	Message: <{"id": "2", "username": "asd", "password": "asd", "porta": "7775"}> sended to the server correctly
2018-10-17 16:49:29	Received a message from the SERVER: {"id": "?", "status": "1"}
2018-10-17 16:49:29	Succesfull logged in as asd
2018-10-17 16:49:29	Received a message from the SERVER: {"id": "4", "messages": {"0": {"Sender": "a", "Text": "asdasd", "Time": "20182018-10-17 16:49:50	Message: <{"id": "3", "username": "asd"}> sended to the server correctly
2018-10-17 16:49:50	Received a message from the SERVER: {"id": "!", "status": "127.0.0.1:7775"}
2018-10-17 16:49:50	Starting a new connection with asd
2018-10-17 16:49:50	asd has IP:Port : 127.0.0.1:772018-10-17 16:50:03	Message: <{"id": "3", "username": "a"}> sended to the server correctly
2018-10-17 16:50:03	Received a message from the SERVER: {"id": "!", "status": "127.0.0.1:58603"}
2018-10-17 16:50:03	Starting a new connection with a
2018-10-17 16:50:03	a has IP:Port : 127.0.0.1:58603
eceived a message from the SERVER: {"id": "!", "status": "-2"}
2018-10-17 16:49:57	Error: you can not connect with yourself
